facebook-common/src/main/java/com/facebook/login/LoginManager.java

```diff
@@ -368,6 +378,7 @@ public void logInWithReadPermissions(Activity activity, Collection<String> permi
         validateReadPermissions(permissions);
 
         LoginClient.Request loginRequest = createLoginRequest(permissions);
+        loginRequest.setAuthType(authType);
```

Comment:
I think it makes more sense to remove these and pass in 'authType' into the Request creation in line 493.